cumulusci.yml

```diff
@@ -67,10 +67,16 @@ tasks:
         options:
             path: unpackaged/config/delete
 
+    robot:
+        options:
+            suites: robot/Cumulus/tests
+            options:
+                outputdir: robot/Cumulus/results
```

Comment:
yay, I've been wanting this. Please do go ahead and add this folder to .gitignore.